src/ogr/geometry.jl

```diff
@@ -1215,42 +1224,70 @@ function getgeom(geom::AbstractGeometry, i::Integer)::IGeometry
     # geometry within the container. The returned geometry remains owned by the
     # container, and should not be modified. The handle is only valid until the
     # next change to the geometry container. Use OGR_G_Clone() to make a copy.
-    if geom.ptr == C_NULL
-        return IGeometry()
-    end
+    geom.ptr == C_NULL && return IGeometry{wkbUnknown}()
     result = GDAL.ogr_g_getgeometryref(geom.ptr, i)
-    if result == C_NULL
-        return IGeometry()
-    else
-        return IGeometry(GDAL.ogr_g_clone(result))
-    end
+    result == C_NULL && return IGeometry{wkbUnknown}()
+    return IGeometry(GDAL.ogr_g_clone(result))
 end
 function getgeom(
```

Comment:
This function can be removed now right?

Reply:
Actually not it turns out. But I don't know why `getgeom` works differently for points? is a `wkbPoint` never actually used anywhere?